test/xskbench/xskbench.c

```diff
@@ -1461,6 +1504,16 @@ PrintUsage(
     ABORT(HELP);
 }
 
+INT64
+UsToQpc(
+    _In_ INT64 Us
+    )
+{
+    LARGE_INTEGER FreqQpc;
+    VERIFY(QueryPerformanceFrequency(&FreqQpc));
```

Comment:
Should this be cached? The doc clearly state that it can, but isn't very clear about if it should (but insistent enough it feels like it)

Reply:
Called only once anyway, so doesn't matter much though.

Reply:
Correct, it can be cached, but xskbench is smarter and all variables on the hot data path use the QPC frequency as the base unit.